### PR TITLE
fixed multiple typos inside docs

### PR DESCRIPTION
**Pull request by parikshitadhikari**
Multiple typos on 6 different files have been fixed inside /docs

Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Comment by parikshitadhikari** [comment]
hey @LefterisJP  , here I have combined all the typos into single pr

**Comment by LefterisJP** [comment]
> hey @LefterisJP , here I have combined all the typos into single pr

Hey thanks. But I specifically asked to not open a new PR but just reuse one of the 6 you opened. You should have just amended the commit of one of them.



Anyway I will review this now.

**Comment by parikshitadhikari** [comment]
sorry, bit of a misunderstanding. anyways, thanks

File: docs/api.rst [hunk 1]
```diff
@@ -3875,7 +3875,7 @@ Refreshing asset icons
 .. http:patch:: /api/(version)/assets/icon/modify
 
    Doing a PATCH on the asset icon endpoint will refresh the icon of the given asset.
-   First, the cache of the icon of the given asset is deleted and then requeried from CoinGecko and saved to the filesystem.
+   First, the cache of the icon of the given asset is deleted and then required from CoinGecko and saved to the filesystem.
```
Comment (LefterisJP) [inline docs/api.rst:3878]: This is wrong. It's not required. It's re-queried.

```suggestion
   First, the cache of the icon of the given asset is deleted and then re-queried from CoinGecko and saved to the filesystem.
```

File: docs/api.rst [hunk 2]
```diff
@@ -3899,7 +3899,7 @@ Refreshing asset icons
 
       {"result": true, "message": ""}
 
-   :statuscode 200: Icon successfully deleted and requeried.
+   :statuscode 200: Icon successfully deleted and required.
```
Comment (LefterisJP) [inline docs/api.rst:3902]: same here

```suggestion
   :statuscode 200: Icon successfully deleted and re-queried.
```

File: docs/changelog.rst [hunk 3]
```diff
@@ -697,10 +697,10 @@ Changelog
...
 * :bug:`2846` Premium users who create a new account with premium api credentials that have no saved DB in the server to sync with will have these credentials properly saved in the DB right after creation. At re-login the premium subscription should be properly recognized and the credentials should not need to be input again.
 * :bug:`2821` Users will now be able to properly scroll through the asset when conflicts appear during the asset database upgrade.
-* :bug:`2837` Binance US users will now be able to see the correct location for their trades and deposits/withdrawals. It should no longer be Binance. To reflect those changes Binance US data should be purged and then requeried. To see how to purge data for an exchange look here: https://rotki.readthedocs.io/en/latest/usage_guide.html#purging-data
+* :bug:`2837` Binance US users will now be able to see the correct location for their trades and deposits/withdrawals. It should no longer be Binance. To reflect those changes Binance US data should be purged and then required. To see how to purge data for an exchange look here: https://rotki.readthedocs.io/en/latest/usage_guide.html#purging-data
```
Comment (LefterisJP) [inline docs/changelog.rst:700]: Same mistake here

```suggestion
* :bug:`2837` Binance US users will now be able to see the correct location for their trades and deposits/withdrawals. It should no longer be Binance. To reflect those changes Binance US data should be purged and then re-queried. To see how to purge data for an exchange look here: https://rotki.readthedocs.io/en/latest/usage_guide.html#purging-data
```

**Review by LefterisJP — COMMENTED** [review]
All right thanks. There is some mistakes. Can you fix them?

File: docs/changelog.rst (hunk 3 above)
Comment (parikshitadhikari) [reply docs/changelog.rst]: done

File: docs/api.rst (hunk 1 above)
Comment (parikshitadhikari) [reply docs/api.rst]: done

File: docs/api.rst [hunk 4]
```diff
@@ -2478,7 +2478,7 @@ Request transactions event decoding
    .. note::
       This endpoint can also be queried asynchronously by using ``"async_query": true``
 
-   Doing a PUT on the evm transactions endpoint will request a decoding of the given transactions and generation of decoded events. That basically entails querying the transaction receipts for each transaction hash and then decoding all events. If events are already queried and ignore_cache is true they will be deleted and requeried.
+   Doing a PUT on the evm transactions endpoint will request a decoding of the given transactions and generation of decoded events. That basically entails querying the transaction receipts for each transaction hash and then decoding all events. If events are already queried and ignore_cache is true they will be deleted and required.
```
Comment (LefterisJP) [inline docs/api.rst:2481]: you still have the same mistake here
```suggestion
   Doing a PUT on the evm transactions endpoint will request a decoding of the given transactions and generation of decoded events. That basically entails querying the transaction receipts for each transaction hash and then decoding all events. If events are already queried and ignore_cache is true they will be deleted and re-queried.
```

File: docs/api.rst [hunk 5]
```diff
@@ -2502,7 +2502,7 @@ Request transactions event decoding
 
    :reqjson list data[optional]: A list of data to decode. Each data entry consists of an ``"evm_chain"`` key specifying the evm chain for which to decode tx_hashes and a ``"tx_hashes"`` key which is an optional list of transaction hashes to request decoding for in that chain. If the list of transaction hashes is not passed then all transactions for that chain are decoded. Passing an empty list is not allowed.
    :reqjson bool async_query: Boolean denoting whether this is an asynchronous query or not
-   :reqjson bool ignore_cache: Boolean denoting whether to ignore the cache for this query or not. This is always false by default. If true is given then the decoded events will be deleted and requeried.
+   :reqjson bool ignore_cache: Boolean denoting whether to ignore the cache for this query or not. This is always false by default. If true is given then the decoded events will be deleted and required.
```
Comment (LefterisJP) [inline docs/api.rst:2505]: same here

**Review by LefterisJP — COMMENTED** [review]
The same mistaken fix is still there in 2 places.

**Comment by parikshitadhikari** [comment]
done

**Review by LefterisJP — APPROVED** [review]
All right all good now. Thank you @p

**Pull request merged**